types/react-native/index.d.ts

```diff
@@ -1,11 +1,12 @@
-// Type definitions for react-native 0.47
+// Type definitions for react-native 0.48
 // Project: https://github.com/facebook/react-native
 // Definitions by: Eloy Durán <https://github.com/alloy>
 //                 Fedor Nezhivoi <https://github.com/gyzerok>
 //                 HuHuanming <https://github.com/huhuanming>
 //                 Kyle Roach <https://github.com/iRoachie>
 //                 Tim Wang <https://github.com/timwangdev>
 //                 Kamal Mahyuddin <https://github.com/kamal>
+//		   Adrian Hall <https://github.com/adrianhall>
```

Comment:
Adding your name here means that you become a maintainer and all issues and pull requests will be sent to your email as well. Just making sure you know what this means.